po/fr.po

```diff
@@ -1,6 +1,5 @@
 # French translation for Gcolor3.
 # Copyright (C) 2013-2015 Nicolas Gaulard-Querol <nicolas.gquerol@gmail.com>
-# Copyright (C) 2015-2016 Timothée Floure <timothee.floure@fnux.ch>
```

Comment:
Looks like you don't like me ;(

It doesn't make any sense, can you fix it ?

Reply:
forgive me i tk accidentally deleted it. well i will correction